spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinProperties.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2012-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.actuate.autoconfigure.tracing.zipkin;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for {@link ZipkinAutoConfiguration}.
+ *
+ * @author Moritz Halbritter
+ * @since 3.0.0
+ */
+@ConfigurationProperties("management.tracing.zipkin")
```

Comment:
Sorry, I can't remember if we already discussed this, but should it be `management.zipkin.tracing` to align with the changes made in https://github.com/spring-projects/spring-boot/issues/30381 on the metrics side of things?

Reply:
Yep, you're right. Haven't updated the branch after #30381, will do this.